_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -150,18 +195,21 @@ Heading marked up with `h` element that does not describe the topic or purpose o
 
 #### Failed example 5
 
-Empty heading marked up with `h` element.
+Heading marked up with `h` element where the visible text nodes describe the content, but the accessible name doesn't.
```

Comment:
Not sure if I should be flagging this stuff but it seems like you are using a kind of short form of English
Heading marked up with `h` element...
as opposed to 
A heading is marked up with an `h` element...

Reply:
It was there in the rule when I started editing it. I'll change it if the native English speaker says so :)